install.sh

```diff
@@ -36,9 +36,12 @@ while true; do
       shift
       ;;
     -h)
-      if [[ -n "$2" ]] && [[ "${2:0:1}" != "-" ]]; then
+      if [[ -n "$2" ]] && [[ "$2" != "" ]] && [[ "${2:0:1}" != "-" ]]; then
         hash="$2"
         shift 2
+      elif [[ "$2" == "" ]]; then
+        hash=""
+        shift 2
```

Comment:
explanation for reviewer:

`-n` checks whether an argument exists, and `"$2"` is the second argument passed in. Here, the argument `"$1"` is the flag that's passed and `"$2"` is the argument associated with it. The first `if` block checks whether such an arg exists, that it's not the explicit empty string `""`, and that it doesn't being with a hyphen, which would indicate a new flag. The second block then checks whether the explicit empty string _is_ passed in.